es-app/src/views/gamelist/BasicGameListView.h

```diff
@@ -11,9 +11,9 @@ class BasicGameListView : public ISimpleGameListView
 	BasicGameListView(Window* window, FileData* root);
 
 	// Called when a FileData* is added, has its metadata changed, or is removed
-	virtual void onFileChanged(FileData* file, FileChangeType change);
+	virtual void onFileChanged(FileData* file, FileChangeType change) override;
 
-	virtual void onThemeChanged(const std::shared_ptr<ThemeData>& theme);
```

Comment:
Compiler warining:

In file included from /home/devel/temp/EmulationStation_Stable/es-app/src/views/gamelist/BasicGameLis
tView.cpp:1:
In file included from /home/devel/temp/EmulationStation_Stable/es-app/src/views/gamelist/BasicGameLis
tView.h:6:
/home/devel/temp/EmulationStation_Stable/es-app/src/views/gamelist/ISimpleGameListView.h:23:15: warni
ng: 'onThemeChanged' overrides a member function but is not marked 'override' [-Winconsistent-missing
-override]
        virtual void onThemeChanged(const std::shared_ptr<ThemeData>& theme);
                     ^
/home/devel/temp/EmulationStation_Stable/es-app/src/views/gamelist/IGameListView.h:27:15: note: overr
idden virtual function is here
        virtual void onThemeChanged(const std::shared_ptr<ThemeData>& theme) = 0;
                     ^